### ISSUE TITLE
can kfac be used with adam at the same time?

### ISSUE THREAD

**Issue by mushan09** [tensorflow/kfac#25]
I find this description in docs:
"K-FAC can be used in place of SGD, Adam, and other Optimizer implementations. However it is slightly more restrictive compared to SGD, Adam as it makes some assumptions on the structure of the model and the loss function."

Now, i want to know if this optimizer can be used with adam at the same time? I don't quite understand the role of the parameter momentum_type.

**Comment by james-martens** [comment]
No, it can't.  Momentum type refers to the type of momentum used within K-FAC.  Momentum is the name of an idea that can be used in many optimizers.  It isn't an "optimizer" itself.  The "adam" type of momentum is just a type of momentum more similar to the type used in Adam than classical momentum methods.
